src/routes/(app)/admin[[hash=admin_hash]]/+page.server.ts

```diff
@@ -1,13 +1,54 @@
-import { CUSTOMER_ROLE_ID } from '$lib/types/User.js';
-import { error, redirect } from '@sveltejs/kit';
+import fs from 'fs';
+import { join } from 'path';
+import { redirect, error } from '@sveltejs/kit';
+import { z } from 'zod';
 import { adminPrefix } from '$lib/server/admin';
+import { CUSTOMER_ROLE_ID } from '$lib/types/User';
+import { runtimeConfig } from '$lib/server/runtime-config';
+import { rootDir } from '$lib/server/root-dir.js';
 
-export async function load({ locals }) {
+export async function load({ url, locals }) {
 	if (!locals.user) {
 		throw redirect(303, `${adminPrefix()}/login`);
 	}
 
 	if (locals.user.roleId === CUSTOMER_ROLE_ID) {
 		throw error(403, 'This page is only accessible to administrators');
 	}
+
+	const docsRootPath = join(rootDir, 'docs');
+	const availableLangs = fs
+		.readdirSync(docsRootPath, { withFileTypes: true })
+		.filter((entry) => entry.isDirectory())
+		.map((entry) => entry.name);
```

Comment:
Should move over this regex too: `const validFolderNameRegex = /^[a-z]{2}(-[a-z]{2})?$/i;`